fakedata.cabal

```diff
@@ -690,6 +690,7 @@ test-suite fakedata-test
   hs-source-dirs:
       test
   ghc-options: -threaded -rtsopts -with-rtsopts=-N
+  build-tool-depends: hspec-discover:hspec-discover
```

Comment:
Is it possible to specificy it via `package.yaml` ? We generate the cabal file via that.